extensions/MYTAditya/CommentPlus.js

```diff
@@ -0,0 +1,376 @@
+                defaultValue: "comment",
+              },
+            },
+          },
+          {
+            opcode: "commentReporterPlusPlus",
+            blockType: Scratch.BlockType.REPORTER,
+            text: "[INPUT] // [COMMENT]",
+            arguments: {
+              COMMENT: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "comment",
+              },
+              INPUT: {
+                type: Scratch.ArgumentType.BOOLEAN,
+              },
+            },
+          },
+          {
+            opcode: "commentBooleanPlusPlus",
+            blockType: Scratch.BlockType.BOOLEAN,
+            text: "[INPUT] // [COMMENT]",
+            arguments: {
+              COMMENT: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "comment",
+              },
+              INPUT: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "",
+              },
+            },
+          },
+          {
+            opcode: "commentReporterP6",
+            blockType: Scratch.BlockType.REPORTER,
+            text: "[COLOR] // [COMMENT]",
+            arguments: {
+              COMMENT: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "comment",
+              },
+              COLOR: {
+                type: Scratch.ArgumentType.COLOR,
+                defaultValue: "#037b03",
+              },
+            },
+          },
+          {
+            opcode: "commentBooleanP6",
+            blockType: Scratch.BlockType.BOOLEAN,
+            text: "[COLOR] // [COMMENT]",
+            arguments: {
+              COMMENT: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "comment",
+              },
+              COLOR: {
+                type: Scratch.ArgumentType.COLOR,
+                defaultValue: "#037b03",
+              },
+            },
+          },
+          {
+            opcode: "commentReporterP7",
+            blockType: Scratch.BlockType.REPORTER,
+            text: "// [COMMENT] [COLOR]",
+            arguments: {
+              COMMENT: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "comment",
+              },
+              COLOR: {
+                type: Scratch.ArgumentType.COLOR,
+                defaultValue: "#037b03",
+              },
+            },
+          },
+          {
+            opcode: "commentBooleanP7",
+            blockType: Scratch.BlockType.BOOLEAN,
+            text: "// [COMMENT] [COLOR]",
+            arguments: {
+              COMMENT: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "comment",
+              },
+              COLOR: {
+                type: Scratch.ArgumentType.COLOR,
+                defaultValue: "#037b03",
+              },
+            },
+          },
+          {
+            opcode: "commentCPlusPlus",
```

Comment:
Can you specify what the point of a double-branch comment block is? As stated previously, the second branch doesn't even run because of how you return the function.

Reply:
Yeah, that block is just pointless, but I created that for the blocks put in first branch would work (like previous ones) and next branch wouldn't work. If you can, please write and comment me the code where both branches would work in the same block.

Reply:
I'm not going to write this extension for you as a block like that shouldn't exist to begin with